packages/manager/modules/billing/src/payment/fidelity/billing-fidelity.html

```diff
@@ -2,161 +2,186 @@
-                    <td class="text-center">
-                        {{ task.balance }}
-                        <span data-translate="fidelity_points"></span>
-                    </td>
-                </tr>
-            </tbody>
-        </table>
+                        </oui-calendar>
+                    </oui-field>
+                </div>
+            </div>
+        </div>
     </div>
+
     <div
-        data-pagination-front
-        data-ng-if="fidelityAccount"
-        data-items="tasksId"
-        data-paginated-items="tasksDetails"
-        data-current-page="currentPage"
-        data-nb-pages="nbPages"
-        data-items-per-page="itemsPerPage"
-        data-transform-item="transformItem(item)"
-        data-on-transform-item-get-promise="getPromise(promise)"
-        data-on-transform-item-notify="onTransformItemNotify(item)"
-        data-on-transform-item-done="onTransformItemDone(items)"
-        data-page-placeholder="{{ 'pagination_page' | translate:{ t0: currentPage, t1: nbPages } }}"
-        data-item-per-page-placeholder="{{ 'pagination_display' | translate }}"
-    ></div>
+        class="table-responsive mt-2"
+        data-ng-if="$ctrl.fidelityAccount && !$ctrl.fidelityLoading"
+    >
+        <oui-datagrid
+            data-rows-loader="$ctrl.getMovements($config)"
+            data-row-loader="$ctrl.getMovementDetails($row)"
+            data-page-size="$ctrl.itemsPerPage"
+        >
+            <oui-datagrid-column
+                title=":: 'fidelity_header_date' | translate"
+                property="date"
+                type="datetime"
+            >
+                <span data-ng-bind="$row.date | date:'mediumDate'"></span>
+            </oui-datagrid-column>
+            <oui-datagrid-column
+                title=":: 'fidelity_header_order' | translate"
+                property="order"
+                type="string"
+            >
+            </oui-datagrid-column>
+            <oui-datagrid-column
+                title=":: 'fidelity_header_operation' | translate"
+                property="description"
+                type="string"
+            >
+            </oui-datagrid-column>
+            <oui-datagrid-column
+                title=":: 'fidelity_header_credit' | translate"
+                property="date"
+                type="datetime"
+            >
+                <span data-ng-show="$row.previousBalance < $row.balance">
+                    <span data-ng-bind="$row.amount"></span>
+                    <span data-translate="fidelity_points"></span
+                ></span>
+                <span data-ng-hide="$row.previousBalance < $row.balance"
+                    >-</span
+                >
+            </oui-datagrid-column>
+            <oui-datagrid-column
+                title=":: 'fidelity_header_debit' | translate"
+                property="date"
+                type="datetime"
+            >
+                <span data-ng-show="task.previousBalance < task.balance"
```

Comment:
Not possible to use a `ng-if` ?

Reply:
Copy paste ^^. I'll check to use `ng-if`